hibernate-core/src/main/java/org/hibernate/boot/spi/MetadataBuildingOptions.java

```diff
@@ -243,5 +243,15 @@ default boolean isXmlMappingEnabled() {
 	/**
 	 * Check to see if extensions can be hosted in CDI
 	 */
-	boolean disallowExtensionsInCdi();
+	boolean isAllowExtensionsInCdi();
```

Comment:
This is an extremely awkward-sounding name. 

How about something like `isCdiIntegrationEnabled()`. WDYT, @sebersole?

Reply:
> This is an extremely awkward-sounding name.
> 
> How about something like `isCdiIntegrationEnabled()`. WDYT, @sebersole?

Please keep in mind `AvailableSettings.ALLOW_EXTENSIONS_IN_CDI` is public, consistency or compatibility will be broken if renaming is made again.
https://github.com/hibernate/hibernate-orm/blob/0897a90a9e674d1a56633622fcd6a240d7dbda41/hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java#L1356